src/openms/source/CHEMISTRY/ProteaseDB.cpp

```diff
@@ -105,4 +105,13 @@ namespace OpenMS
     }
   }
 
+  void ProteaseDB::writeCSV(String const& filename)
+  {
+    std::ofstream ofs(filename, std::ofstream::out);
+    ofs << "OpenMS_AllowedEnzymes" << "\n";
```

Comment:
Just "Enzyme"?

Reply:
Want to make a clear distinction between header and options.